core/src/main/java/bisq/core/provider/mempool/MempoolService.java

```diff
@@ -96,26 +96,36 @@ public void validateOfferMakerTx(OfferPayload offerPayload, Consumer<TxValidator
     }
 
     public void validateOfferMakerTx(TxValidator txValidator, Consumer<TxValidator> resultHandler) {
-        if (!isServiceSupported()) {
-            UserThread.runAfter(() -> resultHandler.accept(txValidator.endResult("mempool request not supported, bypassing", true)), 1);
-            return;
+        if (txValidator.getIsFeeCurrencyBtc() != null && txValidator.getIsFeeCurrencyBtc()) {
+            if (!isServiceSupported()) {
+                UserThread.runAfter(() -> resultHandler.accept(txValidator.endResult("mempool request not supported, bypassing", true)), 1);
+                return;
+            }
+            MempoolRequest mempoolRequest = new MempoolRequest(preferences, socks5ProxyProvider);
+            validateOfferMakerTx(mempoolRequest, txValidator, resultHandler);
+        } else {
+            // using BSQ for fees
+            UserThread.runAfter(() -> resultHandler.accept(txValidator.validateBsqFeeTx(true)), 1);
```

Comment:
This BSQ fee check should be gated by a call to `isServiceSupported()`

Reply:
With the change to have BSQ fees checked using the DAO database rather than with data obtained from mempool.space, `MempoolService` is incorrectly named.  (_In fact, it was unfortunate that it was named referring to an implementation feature rather than functionality_).  

It would be more appropriately called `TradeFeeCheckingService`.  Fee checking should be gated by `isServiceSupported()`, so that the filter and config settings can do their job in enabling/disabling the fee checking feature.